py/damas.py

```diff
@@ -50,7 +52,10 @@ def create( self, keys ) :
             headers=headers, verify=False)
         if r.status_code == 201 or r.status_code == 207:
             return json.loads(r.text)
-        return None
```

Comment:
We would like to return every status codes, so that a Python client could process any status code coming from the server

Reply:
Yes, I am working on it.Once you verify the above.

Reply:
you can use a return value technique here, so that every function would return a tuple, containing the error status code (given by the server), and the data:
```py
return status, data
```
using this technique for both successes and failures, would ease a lot the process of the result on the client side.